site/profile/manifests/core/x2go_agent.pp

```diff
@@ -0,0 +1,20 @@
+# @summary
+#   Adds additional packages for x2go agent
+
+class profile::core::x2go_agent {
+
+  $packages = [
+    'x2goserver',
+    'x2goclient',
+    'x2goserver-common',
+    'x2goserver-xsession',
+    'x2goagent',
+  ]
+
+  package { $packages:
+    ensure => 'present',
+  }
+  yum::group { 'Mate':
```

Comment:
I see that there is a `mate` mod in the `Puppetfile`. Can we use that to install the mate desktop?

Reply:
I'll give it a go.